html-forms/index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+  <h1>Coding Form!</h1>
+  <form action="index.html" method="get">
+    <div>
+      <label for="email">Email</label>
+    </div>
+    <div>
+      <input type="text" id="email" name="email">
+    </div>
+    <div>
+      <label for="whycoding">Why coding is cool?</label>
+    </div>
+    <div>
+      <select name="whycoding" id="whycoding">
+        <option value="Select an option">Select an option</option>
+        <option value="Because it's interesting">Because it's interesting</option>
+        <option value="Because it's fun">Because it's fun</option>
+        <option value="Because it's powerful">Because it's powerful</option>
+      </select>
+    </div>
+    <div>
+      <label for="favorite_language">Favorite Language (Choose one)</label>
+    </div>
+    <div>
+      <input type="radio" name="favorite_language" value="HTML">HTML
+      <input type="radio" name="favorite_language" value="CSS">CSS
+      <input type="radio" name="favorite_language" value="JavaScript">JavaScript
+    </div>
+    <div>
+      <label for="signup">Would you like to sign up for more information?</label>
+    </div>
+    <div>
+      <input type="checkbox" name="signup" value="signup">Sign me up!
```

Comment:
"Sign me up!" should be a `label` element that is properly configured to match with the `input`